Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_hammer.yml

```
@@ -0,0 +1,33 @@
+- type: entity
+  parent: BaseItem
+  id: ArmHammerChangeling
+  name: arm hammer
+  description: A hammer made out of flesh and bone, heavy enough to smash through structures and people alike.
+  suffix: Unremoveable
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Changeling/hammer.rsi
+    state: icon
+  - type: MeleeWeapon
+    angle: 0
+    wideAnimationRotation: -90
+    attackRate: 0.5
+    damage:
+      types:
+        Blunt: 40
+        Structural: 90
```

40 is still wayy too much... 30 would work?
```suggestion
        Blunt: 30
```

---

Its attack rate is 0.5 so DPS is actually lowest amongst the 3 ling weapons.

---

Let's gather 3 people and put em to practical test. I believe claw or hammer will win easily due to the claw DPS and Hammer utility (the knock back is insane)